lib/need_for_type/display_window.rb

```diff
@@ -37,52 +37,63 @@ def standout_mode(option, selected_option)
     end
 
     def render_start_game
-      self.render do
+      self.render_box do
         self.set_render_pos(2, 4)
         self.render_text("To start playing click ENTER", GREEN, NORMAL)
       end
     end
 
-    def render_game_text(text, chars_completed, failed = false)
+    def render_game_text(text, chars_completed, stats, failed = false)
       color = failed ? RED : YELLOW
       completed = text[0, chars_completed] || ''
       remaining = text[chars_completed + 1, text.length] || ''
 
-      @current_line = 2
-      @current_col = 4
+      self.render_box do
+        # Stats
+        self.render_with_color(GREEN) do
+          self.set_render_pos(2, 4)
+          self.addstr("Time: #{stats[:total_time].round(2)} sec")
 
-      self.render do
+          self.set_render_pos(2, 21)
+          self.addstr("WPM: #{stats[:wpm].round}")
+
+          self.set_render_pos(2, 30)
+          self.addstr("Accuracy: #{stats[:accuracy].round(2)} %")
```

Comment:
Can't we use `render_score` (a method made on line 72 for this? just to try not repeating the same logic